package.json

```diff
@@ -19,10 +19,18 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "gulp test && coffee tests/e2e.coffee",
-    "prepublish": "gulp build",
-    "readme": "jsdoc2md --template doc/README.hbs build/imagefs.js > README.md"
+    "lint": "resin-lint lib tests",
+    "build": "coffee --bare -o build lib",
+    "test": "npm run lint && npm run build && coffee tests/e2e.coffee",
+    "readme": "jsdoc2md --template doc/README.hbs build/imagefs.js > README.md",
+    "prepublish": "npm run test && npm run readme"
```

Comment:
Should this also run the `build`?

Reply:
build is already ran by test

Reply:
@jhermsmeier I updated it so test no longer builds and prebublish tests, builds and creates the docs.